tests/ban_test.py

```diff
@@ -49,6 +44,6 @@ def test_unban_cmd(self, mockjson, mockreq):
             {'id': '897', 'uri': 'coala/coala-bears'}
         ]
         mockjson.loads.return_value = fake_room_data
-        testbot.assertCommand('!unban @nvzard',
```

Comment:
`testbot = self` (and similar throughout) will make this patch much smaller, easier to review, allowing reviewers to focus on the more important other changes that are occurring.